js/main.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-console */
+function getNumber (min, max) {
+  const randomNumber = Math.random();
+
+  if (min > max  ) {
+    console.log('Ошибка');
+  }
+
+  if (min <= max && min >= 0) {
+    return Math.floor(randomNumber * (max - min + 1)) + min;
+  }
+}
+getNumber(10, 10);
+
+function getMaxLenghtString(string, maxLenghtString) {
+  if (string.length <= maxLenghtString) {
+    return true;
+  }
+  if (string.length > maxLenghtString) {
+    return false;
+  }
```

Comment:
```suggestion
    return string.length <= maxLenghtString);
```
Это тебе на подумать)
Функция должна вернуть `true/false`, что она и делает в двух вариантах)